Compilation error

```
$ ./build.sh 
pcap2flow.c:19:10: fatal error: pcap/pcap.h: No such file or directory
   19 | #include <pcap/pcap.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
```

`sudo apt-get install libpcap-dev` on ubuntu fixes the above.. but then you run into:

```
pcap2flow.c: In function ‘main’:
pcap2flow.c:131:36: warning: implicit declaration of function ‘atoi’ [-Wimplicit-function-declaration]
  131 |   nf_init_peer(&nf_peer, &peer_ip, atoi(argv[3]));
      |    
```

--- a/README.md
+++ b/README.md
@@ -2,3 +2,6 @@ pcap2netflow
 ============
 
 Program to read a pcap file and export the flows as cflow/netflow records
+
+If you are going to compile this on Ubuntu, you require the `libpcap-dev` package.  You can install this by using:
+`sudo apt-get install libpcap-dev`

--- a/pcap2flow.c
+++ b/pcap2flow.c
@@ -20,6 +20,7 @@
 #include "nf_time.h"
 #include "hash.h"
 #include "flowtable.h"
+#include <stdlib.h>
 
 #define FLOW_TABLE_SIZE 65536
 #define FLOW_TIMER 3600000

